packages/app/src/systems/Transaction/components/TxOperation/TxOperation.tsx

```diff
@@ -0,0 +1,35 @@
+import { cssObj } from '@fuel-ui/css';
+import { Card } from '@fuel-ui/react';
+
+import type { Operation, Status } from '../../utils';
+import { TxFromTo } from '../TxFromTo/TxFromTo';
+
+import { AssetsAmount } from '~/systems/Asset';
+
+export type TxOperationProps = {
+  operation?: Operation;
+  status?: Status;
+};
+
+export function TxOperation({ operation, status }: TxOperationProps) {
```

Comment:
we don't need a loader for this `TxOperation` component as well?

Reply:
yes good catch... I will also go in the screens and apply it.